### PR TITLE
upgrading sveltekit

### PR DESCRIPTION
**Pull request by brittneypostma**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+strict-peer-dependencies=false
```
Comment (ghostdevv) [inline .npmrc:1]: Any reason for this?

**Comment by ghostdevv** [comment]
@brittneypostma There were a few packages that didn't update fully, maybe that will fix the need for that npmrc file. I didn't update skeleton though as it's latest version is `1.8.0` and I don't know enough about it to fix breakages right now - happy to look later though if you can't

**Comment by ghostdevv** [comment]
Great the build broke, I don't think vite preprocess likes the `-` in the class names in style tags - any ideas?

**Comment by brittneypostma** [comment]
The Svelte local storage package is what was causing that. Not sure if I checked the version it is on, but I don't think it's updated.

**Comment by ghostdevv** [comment]
Fixed the build & updated the deps again - I haven't updated skeleton just yet but will try and get round to that at some point

File: svelte.config.js [hunk 2]
```diff
@@ -26,7 +26,11 @@ const config = {
 		}
 	},
 
-	preprocess: [vitePreprocess()]
+	preprocess: [
+		preprocess({
+			postcss: true
```
Comment (spences10) [inline svelte.config.js:31]: Hey @ghostdevv I've been using `vitePreprocess` is there advantages to using `svelte-preprocess` over the Vite one?

Comment (ghostdevv) [reply svelte.config.js:31]: `vitePreprocess` is better but it doesn't seem to like the `:` in the tailwind classes in style blocks, and I don't know any other way to fix it

**Pull request merged**